tests/cbmc/proofs/s2n_stuffer_write_base64/Makefile

```diff
@@ -11,7 +11,7 @@
 # implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Enough to get full coverage with 9 minutes of runtime.
+# Enough to get full coverage with 11 minutes of runtime.
```

Comment:
Per feliperodri, the additional 2 minutes is due to using a new version of CBMC *and* considering more paths since we now assume malloc may fail.